multitenancy/tenant-manager-api/src/main/java/io/apicurio/multitenant/storage/dto/RegistryTenantDto.java

```diff
@@ -91,6 +97,28 @@ public void setOrganizationId(String organizationId) {
         this.organizationId = organizationId;
     }
 
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        if (description != null && description.length() > 512) {
```

Comment:
bug

Reply:
Damn you, copy/paste!  Good catch thanks.